packages/as-transformer/src/transformers/massaExport.ts

```diff
@@ -142,6 +142,8 @@ export class MassaExport {
     })`;
     if (this.returnType && this.returnType !== 'void') {
       wrapper += `  const response = encode${this.functionName}RHelper(new ${this.functionName}RHelper(${call}));\n\n`;
+
+      wrapper += "  generateEvent(`Result:'${response.buffer.toString()}'`);\n";
```

Comment:
Can we use ${this.functionName}Response instead to avoid conflict on matching responses from the front ?

Reply:
I updated the pattern as "Result${this.functionName}:" as requested by @Elli610

Reply:
maybe we should add the first x digits of the contract address to avoid confusions with functions that have the same name and could be called in another contract